oauth2_http/java/com/google/auth/oauth2/ServiceAccountCredentials.java

```diff
@@ -612,14 +612,15 @@ public GoogleCredentials createScoped(Collection<String> newScopes) {
   }
 
   /**
-   * Clones the service account with the specified scopes.
+   * Clones the service account with the specified scopes. The Access Token is invalidated even if
```

Comment:
When this draft is ready for review, would you do:

1. Would you add a sentence about the purpose of invalidate access token?
2. In the pull request description, would you add a rationale `setAccessToken(null)` does not break existing code that use the `createScoped` method?

Reply:
Thanks, added.